run.py

```diff
@@ -14,6 +14,19 @@
 
 import llm
 
+# TODO: this is not the right place to define these things
+min_lines = 5
+expansion_count = None
+
+# TODO: this is not the right place to define check_func
```

Comment:
this is Dafny-specific. Make sure run, run_whole and run_intermediate_expansion work with Coq as well.